versions.json

```diff
@@ -8,7 +8,7 @@
   "nixpkgs": {
     "owner": "flyingcircusio",
     "repo": "nixpkgs",
-    "rev": "a3a114102ca9c55fc95df5a92d4724b13c612c3a",
-    "sha256": "0xnxq2ykgr2qxwrb7dl342l2sb9c9153zr0hpc5x9pm23mcbj7ji"
+    "rev": "0a92f12c2f6462c5843ce325a1d47116e6a030eb",
```

Comment:
Commit is from the branch security-updates, should be from nixos-19.03. Maybe just cherry-pick the 2 security update commits to nixos-19.03?